src/openrct2/world/Entrance.cpp

```diff
@@ -584,32 +584,32 @@ void maze_entrance_hedge_removal(int32_t x, int32_t y, rct_tile_element* tileEle
     x += CoordsDirectionDelta[direction].x;
     y += CoordsDirectionDelta[direction].y;
     int32_t z = tileElement->base_height;
-    int32_t rideIndex = track_element_get_ride_index(tileElement);
+    int32_t rideIndex = tileElement->properties.entrance.ride_index;
```

Comment:
Should this not use the `GetRideIndex()` function?

Reply:
The Entrance element does not yet have this function, AFAIK.